XCUITest/WebsiteAccessTest.swift

```diff
@@ -20,7 +20,7 @@ class WebsiteAccessTests: BaseTestCase {
         searchOrEnterAddressTextField.tap()
         searchOrEnterAddressTextField.typeText("mozilla")
         waitForValueContains(label, value: "mozilla.org/")
-        waitForExistence(app.buttons["Search for mozilla"])
+        // waitForExistence(app.buttons["Search for mozilla"]) Different for refresh branch - change it later
```

Comment:
@bogdancernea28 Can you please file a github issue with the `Visual Refresh` label on it for this so that we don't forget?

Reply:
> @bogdancernea28 Can you please file a github issue with the `Visual Refresh` label on it for this so that we don't forget?

@st3fan done